Team9Project/src/com/indragie/cmput301as1/ExpenseClaimDetailActivity.java

```diff
@@ -225,7 +287,56 @@ public boolean onCreateOptionsMenu(Menu menu) {
 	@Override
 	public boolean onPrepareOptionsMenu(Menu menu) {
 		super.onPrepareOptionsMenu(menu);
-		menu.findItem(R.id.action_add_item).setEnabled(editable);
+		boolean UserCheck = user.getName().contentEquals(claim.getUser().getName());//SHOULD BE ID USING NAME FOR TESTING
```

Comment:
This will need to be changed to ID yes, but because of the lack online functionality I have no way of tracking or implementing incremental ID's yet. So I think for now it should be just left as is to maintain functionality?